web-client/src/presenter/computeds/formattedEligibleCasesHelper.ts

```diff
@@ -43,42 +44,73 @@ const getSortableDocketNumber = docketNumber => {
   return `${year}-${number.padStart(6, '0')}`;
 };
 
-const getFullSortString = (theCase, cases) => {
-  if (!Array.isArray(cases)) {
-    return '';
-  }
-  const leadCase = cases.find(
-    aCase => aCase.docketNumber === theCase.leadDocketNumber,
-  );
-
-  const isLeadInEligible = !!theCase.leadDocketNumber && !!leadCase;
-
-  let priorityPrefix = 'C';
+const getFullSortString = theCase => {
+  let priorityPrefix = '';
 
+  if (theCase.isAgedCase) {
+    priorityPrefix += 'A';
+  }
   if (theCase.isManuallyAdded) {
-    priorityPrefix = 'A';
-  } else if (theCase.isDocketSuffixHighPriority) {
-    priorityPrefix = 'B';
+    priorityPrefix += 'B';
+  }
+  if (theCase.isDocketSuffixHighPriority) {
+    priorityPrefix += 'C';
   }
 
-  return `${priorityPrefix}_${getSortableDocketNumber(
-    isLeadInEligible
-      ? theCase.docketNumber === theCase.leadDocketNumber
-        ? theCase.docketNumber
-        : theCase.leadDocketNumber
-      : theCase.docketNumber,
-  )}-${getSortableDocketNumber(theCase.docketNumber)}`;
+  priorityPrefix += 'D';
+
+  return `${priorityPrefix}_${getSortableDocketNumber(theCase.docketNumber)}`;
 };
 
-export const compareTrialSessionEligibleCases = eligibleCases => {
-  const groups = getPriorityGroups(eligibleCases);
+export const compareTrialSessionEligibleCases = () => {
   return (a, b) => {
-    const aSortString = getFullSortString(a, groups[a[groupKeySymbol]]);
-    const bSortString = getFullSortString(b, groups[b[groupKeySymbol]]);
+    const aSortString = getFullSortString(a);
+    const bSortString = getFullSortString(b);
     return aSortString.localeCompare(bSortString);
   };
 };
 
+export const getFormattedEligibleCases = formattedCases => {
+  const memberCasesObj = {} as Record<string, any[]>;
```

Comment:
I believe we know the type here. Should be `RawEligibleCase` or `Omit<RawEligibleCase, 'whatever' | 'was' | 'omitted'>`